app/controllers/articles_controller.rb

```diff
@@ -109,7 +109,8 @@ def preview
   def create
     authorize Article
 
-    article = Articles::Creator.call(current_user, article_params_json)
+    @user = current_user
+    article = Articles::Creator.call(@user, article_params_json)
```

Comment:
I think this change slipped in, there's no need to store in `@user`

Reply:
I do think this is necessary, as `article_params_json` uses `@user`, but I could be wrong.